charts/rancher-logging/0.1.2/values.yaml

```diff
@@ -4,6 +4,5 @@ fluentd-tester:
   enabled: false
 log-aggregator:
   enabled: false
```

Comment:
Conditions in requirements.yml has changed to `log-aggregator.log-aggregator-<OS>.enabled`, I think we should update them in the values as well

Reply:
this values.yaml file is for the first level sub charts, first level sub charts only have fluentd, fluentd-test, log-aggregator. <OS> is included in the second level sub charts.